src/test/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositAppSpec.scala

```diff
@@ -122,6 +122,42 @@ class SplitMultiDepositAppSpec extends TestSupportFixture with MockFactory with
       app.convert(paths, datamanager) shouldBe a[Success[_]]
     }
 
+    val uuidRegex = "[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}"
+
+    it should "check report.csv" in {
+      doNotRunOnTravis()
+
+      reportFile.toJava should exist
+
+      val header :: lines = reportFile.lines.toList
+      val reportLines = lines.collect {
+        case s if s startsWith ruimtereis01 => ruimtereis01 -> s
+        case s if s startsWith ruimtereis02 => ruimtereis02 -> s
+        case s if s startsWith ruimtereis03 => ruimtereis03 -> s
+        case s if s startsWith ruimtereis04 => ruimtereis04 -> s
+      }.toMap
+      // taken from https://stackoverflow.com/a/6640851/2389405
+      // and https://github.com/DANS-KNAW/easy-split-multi-deposit/pull/111#discussion_r194733478
```

Comment:
Shouldn't this comment belong to the regexp?

Reply:
You're right! 👍
I moved the `uuidRegex` to a different place, for it to be reused later on, but forgot to move the comment(s) with it.
Thanks!

Reply:
a8ff843